woocommerce/woocommerce
# [Product Block Editor]: Add new image components

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR introduce three new image components:

`<Shirt />`, `<Pants />` and `<Glasses />`.





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Let's use Storybook

1. Update, just in case:

```cli
pnpm install
````

2. Run the Storybook app

```cli
pnpm --filter=storybook run storybook
```

3. Go to the Images stories. For instance, the Glasses story

`http://localhost:6007/?path=/story/product-editor-images-glasses--default`

4. Confirm you see the story
5. Play with the component properties

<img width="918" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/d9aa2c31-8a57-41f7-a387-e97f2def1d66">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
